taskt/UI/Forms/Supplement Forms/frmBrowserElementBuilder.cs

```diff
@@ -12,19 +12,14 @@
 //See the License for the specific language governing permissions and
 //limitations under the License.
 using System;
-using System.Collections.Generic;
-using System.ComponentModel;
 using System.Data;
-using System.Drawing;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 using System.Windows.Forms;
+using System.Reflection;
 using SHDocVw;
 using MSHTML;
-using System.Runtime.InteropServices;
+using taskt.Core.Automation.Commands;
 
-namespace taskt.UI.Forms.Supplemental
+namespace taskt.UI.Forms.Supplement_Forms
 {
     public partial class frmBrowserElementBuilder : UIForm
     {
```

Comment:
This is actually for line 30 but github won't let me select it. Don't these variables need access modifiers??

Reply:
That's a good point, I totally glazed over that. If you leave out the access modifiers C# just uses public. Made the changes.